Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1116,11 +1116,16 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
             print_warning('Collecting sample tests due to: {}'.format(','.join(sample_tests)))
         else:
             print_warning("Running Sanity check only")
-            tests.add('DocumentationTest')  # test with integration configured
+
             tests.add('TestCommonPython')  # test with no integration configured
+            packs_to_install.add("DeveloperTools")
```

Comment:
why do you need this? aren't we always installing that pack?

Reply:
Installing but not re-uploading. @yaakovi see my comment on second pr